control/webapp/jobs.py

```diff
@@ -85,6 +85,15 @@ def status(id):
 
     return render_template("jobs/status.html", job=job, for_society=for_society, owner_in_context=owner_in_context, job_home_url=job_home_url, member=mem)
 
+@bp.route('/jobs/<int:id>.json')
+def status_json(id):
+    job = Job.find(sess, id)
+    if not job:
+        raise NotFound(id)
+    if not job.visible_to(utils.raven.principal):
+        raise NotFound(id)
+    return jsonify({"state": job.state})
```

Comment:
Open question - is it worth putting any extra information into this JSON object? (e.g. URL of status page)

Reply:
I wasn't intending for this to be a "public API" in any sense, so I don't think we need to add anything beyond what we want to display in updates.  The job ID and URL are needed before the first status request, so they're baked into attributes in the base template and won't be changing.